samples/browser/src/main/java/org/mozilla/samples/browser/BaseBrowserFragment.kt

```diff
@@ -185,6 +190,24 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler {
             owner = this,
             view = layout)
 
+        p2pIntegration.set(
+                feature = P2PIntegration(
+                        store = components.store,
+                        engine = components.engine,
+                        view = layout.p2p,
+                        thunk = { -> components.nearbyConnection },
```

Comment:
```suggestion
                        thunk = { components.nearbyConnection },
```

You can omit the arrow if there aren't any arguments for the callback.